crates/assistant/src/main.rs

```diff
@@ -22,6 +22,7 @@ use embedded_hal::serial::{Read as SerialRead, Write as SerialWrite};
 
 const UART_DRIVER: Channel = Channel::new(0);
 const TALENT: Channel = Channel::new(1);
+const ETH_TEST: Channel = Channel::new(3);
```

Comment:
Remove `ETH_TEST` references for now, we have a working timer in a different branch. This file should not have any changes,